apis/nucleus/src/components/Footer.jsx

```diff
@@ -28,15 +29,37 @@ const CellFooter = {
   className: 'njs-cell-footer',
 };
 
-function Footer({ layout, titleStyles = {} }) {
-  return layout && layout.showTitles && layout.footnote ? (
+function Footer({ layout, titleStyles = {}, translator, flags, isCardTheme }) {
+  const footerStyle = titleStyles.footer;
+  const hasFilters = layout?.filters?.length > 0 && layout?.qHyperCube?.qMeasureInfo?.length > 0;
+  const filtersFootnoteString = generateFiltersString(layout?.filters ?? [], translator);
```

Comment:
You have these two functions: generaFiltersString and generateFiltersLabels, but one is called out here in Footer and one inside FiltersFooter. I think it would be cleaner if all that is done inside the FiltersFooter.
I see that you need it to check if the FiltersFooter should exist or not, but you can do that check inside the component instead, which would clean up the Footer quite a bit.

Reply:
its ok